solr/core/src/java/org/apache/solr/handler/api/ApiRegistrar.java

```diff
@@ -19,6 +19,8 @@
 
 import org.apache.solr.api.ApiBag;
 import org.apache.solr.handler.admin.CollectionsHandler;
+import org.apache.solr.handler.admin.CoreAdminHandler;
+import org.apache.solr.handler.admin.InfoHandler;
 import org.apache.solr.handler.admin.api.*;
```

Comment:
usual nit pick on .*

Reply:
I did update my code settings!  I guess IntelliJ only rewrites those imports that change as you edit though?